packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/TransactionsCompletedRow.tsx

```diff
@@ -111,7 +141,7 @@ const TransactionsCompletedRow: React.FC<TProps> = ({ accounts, transaction, wal
     return (
         <React.Fragment>
             <Divider color='var(--border-divider)' />
-            <button
+            <div
```

Comment:
Need to change this to div because the `WalletClipboard` component is a button, and nesting buttons is not allow.